### PR TITLE
Add media queries to make website responsive 

### PR DESCRIPTION
**Pull request by bilalrajput09**
1. I added **media queries** to make the website to be compatible with big screens. 

2. Also, I made a few changes in the **mobile version** to make things easier with media queries.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,103 +78,118 @@ <h2 class="intro2">I’am a Software Developer</h2>
...
           <img src="./img/placeholder.png" class="placeholder" alt="image1" />
+          </div>
+          <div class="multistories">
           <h2>Multi-Post Stories</h2>
           <p>
             A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
+            sign-ups required. This has been the industry's standard dummy text ever
             since the 1500s, when an unknown printer took a standard dummy text.
           </p>
```
Comment (leonardodiasb) [inline index.html:93]: - [ ] The main project style is great but the font-family used does not match the Figma template. Kindly use the font 'Inter' as you used on other parts of this project to style the paragraph and title of this project to match the template.

File: style.css [hunk 2]
```diff
@@ -192,19 +206,29 @@ header {
...
-  margin-left: 1.5rem;
-  margin-top: 24px;
-  margin-right: 1.5rem;
   color: white;
+  height: 435px;
+  width: 369px;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
 }
```
Comment (leonardodiasb) [inline style.css:229]: - [ ] Your regular project cards are great. But the 'See Project' button should only appear when the user hovers over the card, making the title, description, technologies list and dark overlay of the image disappear, and the 'See Project' button should appear at the bottom of the image like the Figma template. Kindly implement this feature to match the Figma template.
 - Your design:
 - 
image

 - Figma template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @bilalrajput09, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub workflow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -72,172 +81,258 @@ <h2 class="intro2">I’am a Software Developer</h2>
...
+              sign-ups required. has been the industry's standard
+            </p>
+
+            <ul class="languages-2">
+              <li class="html">Html</li>
+              <li class="css">Css</li>
+              <li class="javascript">Javascript</li>
+            </ul>
+            <a href="#" class="cards-project">See Project</a>
+          </div>
```
Comment (vickymarz) [inline index.html:198]: - [ ] Good job implementing the suggested changes on the card. Kindly ensure that when you hover over any of the project card, the dark overlay disappears thereby showing a clear image on hover state.

Currently, the dark overlay is still on the image


Figma design 👇🏽 
image

File: index.html [hunk 4]
```diff
@@ -27,12 +27,21 @@
...
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><img src="./img/mail.png" alt="mail" /></li>
+        </ul>
       </nav>
+      <img src="./img/threelines.png" class="menu" alt="menu bar" />
     </header>
-    <section>
+    <section class="section-1">
```
Comment (vickymarz) [inline index.html:39]: [OPTIONAL]

- I can see a horizontal scroll bar on the desktop version of your website meaning content is overlapping the website view width which is not a good user experience. Kindly ensure that no horizontal scroll is present on your website. 

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @bilalrajput09, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Shedrack-Sunday) [inline index.html:39]: - [Opional] Though the previous reviewer also made the changes here optional, I also strongly advise that you fix them.  the horizontal scroll bar on the desktop version of your website content is overlapping with the website view width which is not a good user experience. You may want to check your wrapped `CSS` styling concerning this section. 

# View:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @bilalrajput09   👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing some of the features requested by the previous reviwer, However, there are a few you need to work on based on the previous reviewer's suggestion to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 


Required Changes ♻️

- [x] Firstly, you did have a very beautiful design. however, the goal is to stick to the recommended one just as it will be in your career as a developer. I've made a detailed review of sections that needs fixing. kindly follow the suggestions, and you'll be approved in no time. Excellent work you did 👏 

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

- [ ] Your wrapper width seems to be overlapping with the screen size. though optional, i made a detailed review below to help you with that. 


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 4 above)
Comment (bilalrajput09) [reply index.html:39]: Thanks for the review I tried but things messing up with mobile version code I have no idea how to solve the problem can you please check where I am making mistake ?

Comment (bilalrajput09) [reply index.html:39]: hey I have fixed the issue

**Review by Tadesse-Alemayehu — APPROVED** [review]
### Hi @bilalrajput09 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f692e67696665722e636f6d2f6f726967696e2f36352f36353164353034326534396532656561363365613934633039653131393731365f773230302e676966


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -532,3 +641,253 @@ footer {
...
+  }
+
+  .about-me > h1 {
+    font-size: 72px;
+  }
+
+  .about-me > p {
+    font-size: 20px;
+    width: 565px;
+  }
```
Comment (Qoosim) [inline style.css:833]: - [OPTIONAL] The right image on the background image used does not fall at the right place on your design compare to how it is on the Figma Design. Kindly look into it and fix.

### Your Design
your-about-logo

### Figma Design
temp-about-logo

File: style.css [hunk 6]
```diff
@@ -112,13 +123,44 @@ header {
...
+  margin: 1.5rem;
+}
+
+.multistories > h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+}
+
```
Comment (Qoosim) [inline style.css:158]: - [OPTIONAL] There's no consistency in the margins(right and left) maintained throughout all the projects under recent works sections. The width used for the first project is wider as opposed to the rest of the projects. And this shows a great difference in your design compare to the Figma design. Kindly take a look, compare and work your design alongside the Figma design.

### Your Design
your-projects-consistency

### Figma Design
temp-projects-spacing

File: style.css [hunk 7]
```diff
@@ -112,13 +123,44 @@ header {
...
+.placeholder {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.recent-work {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(1, 1fr);
+}
```
Comment (Qoosim) [inline style.css:143]: - [OPTIONAL] Kindly reduce the `margin-top` between the recent work title and the first project. Take a look and fix it.

### Your Design
your-work-spacing

### Figma Design
temp-work-area

**Review by Qoosim — COMMENTED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @bilalrajput09, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-   Fair README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**